packages/story-editor/src/components/panels/panel/shared/content.js

```diff
@@ -33,7 +33,7 @@ const Container = styled.div`
     isSecondary ? theme.colors.interactiveBg.secondaryNormal : 'transparent'};
 `;
 
-function Content({ children, ...rest }) {
+function Content({ children, isPrimary = false, ...rest }) {
```

Comment:
Not sure what this param is doing or if it is needed?

Reply:
No, it was changed to `isSecondary` (i.e. flipped, so the default undefined would serve as false just fine), so it can be removed completely (from propTypes as well).

Reply:
Fixed in https://github.com/GoogleForCreators/web-stories-wp/pull/12949/commits/2d4c4869329dbdde54406f324425884044c028de